test/uk/gov/hmrc/eusubsidycompliancefrontend/test/CommonTestData.scala

```diff
@@ -158,6 +158,15 @@ object CommonTestData {
     List(businessEntity4)
   )
 
+  val undertaking3 = Undertaking(
+    undertakingRef,
+    UndertakingName("TestUndertaking"),
+    agriculture,
+    IndustrySectorLimit(32.34).some,
```

Comment:
One small comment which is entirely preference, for me Some(LocaDate etc) Reads better then adding .some on the end. Again preference so happy to go with what reads better for all!

Reply:
Thanks Jack , I've made the change

Reply:
nice! as long as it reads well for all thanks for changing!